drivers/SmartThings/zigbee-window-treatment/profiles/window-treatment-screeninnovations.yaml

```diff
@@ -0,0 +1,21 @@
+name: window-treatment-screeninnovations
+components:
+  - id: main
+    capabilities:
+      - id: windowShade
+        version: 1
+      - id: windowShadeLevel
+        version: 1
+      - id: windowShadePreset
+        version: 1
+      - id: battery
+        version: 1
+      - id: refresh
+        version: 1
+      - id: powerSource
```

Comment:
@greens - Should an embedded profile be considered here? To list what types of powerSource it supports?

Reply:
they're relying on the default handler here, which can return all possible values

if they know they won't be sending some of them, then it might be a good idea